dsf-bpe/dsf-bpe-process-base/src/main/java/org/highmed/dsf/bpe/Constants.java

```diff
@@ -56,6 +51,7 @@
 	String CODESYSTEM_HIGHMED_FEASIBILITY_VALUE_NEEDS_RECORD_LINKAGE = "needs-record-linkage";
 	String CODESYSTEM_HIGHMED_FEASIBILITY_VALUE_SINGLE_MEDIC_RESULT = "single-medic-result";
 	String CODESYSTEM_HIGHMED_FEASIBILITY_VALUE_PARTICIPATING_MEDICS_COUNT = "participating-medics";
+	String CODESYSTEM_HIGHMED_FEASIBILITY_VALUE_NOT_ENOUGH_PARTICIPATION = "not-enough-participation";
 	String CODESYSTEM_HIGHMED_FEASIBILITY_VALUE_MULTI_MEDIC_RESULT = "multi-medic-result";
 	String CODESYSTEM_HIGHMED_FEASIBILITY_VALUE_RESEARCH_STUDY_REFERENCE = "research-study-reference";
 
```

Comment:
Long term we might want to move process specific constants to the respective process modules

Reply:
Should we wait with moving the constants until we included the data sharing process? It could be that we reuse some of the constants for this process as well.

Reply:
Ja, that's what I thought, lets see what constants we want to reuse. I'll create an issue to track this.